contracts/Main.sol

```diff
@@ -65,25 +64,49 @@ contract Main {
         newTRBContract.mint(msg.sender, balance);
     }
 
-    // Helper function for contracts with public admin to
-    // send the funds directly to its address.
-    //slither-disable-next-line unimplemented-functions
-    function migrateContract(address _contract) public onlyAdmin {
-        address _owner = Owned(_contract).owner();
-        migrateContractTo(_contract, _owner);
-    }
-
     //slither-disable-next-line unimplemented-functions
     function migrateContractTo(address _contract, address _owner)
         public
         onlyAdmin
     {
+        _migrateContractTo(_contract, _owner);
+    }
+
+    function migrateContractToBatch([]address calldata _contracts,  []address calldata _owners) public onlyAdmin {
```

Comment:
Batch functions are helpful because they become cheaper than doing multiple calls. Since it's the admin calling, might be worth it to save us a couple of bucks